lib/ansible/plugins/inventory/tower.py

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: tower
+    plugin_type: inventory
+    authors:
+      - Matthew Jones (@matburt)
+      - Yunfan Zhang (@YunfanZhang42)
+    short_description: Ansible dynamic inventory plugin for Ansible Tower.
+    description:
+        - Imports inventories from Ansible Tower.
+        - Supports reading configuration from both YAML config file and environment variables.
+        - If reading from the YAML file, the file name must end with tower_inventory.(yml|yaml),
+          the path in the command would be /path/to/tower_inventory.(yml|yaml). If some arguments in the config file
+          are missing, this plugin will try to fill in missing arguments by reading from environment variables.
+        - If reading configurations from environment variables, the path in the command would be tower_inventory.
+    options:
+        tower_host:
+            description: The network address of your Ansible Tower host.
+            type: string
+            env:
+                - name: TOWER_HOST
+            required: True
+        tower_username:
+            description: The user that you plan to use to access inventories on tower.
+            type: string
+            env:
+                - name: TOWER_USERNAME
+            required: True
+        tower_password:
+            description: The password for tower_username.
+            type: string
+            env:
+                - name: TOWER_PASSWORD
+            required: True
+        tower_inventory:
+            description: The ID of the tower inventory you wish to import.
+            type: string
+            env:
+                - name: TOWER_INVENTORY
+            required: True
+        tower_verify_ssl:
+            description: Specify whether Ansible should verify the SSL certificate of Ansible tower.
+            type: bool
+            default: True
+            env:
+                - name: TOWER_VERIFY_SSL
+            required: False
```

Comment:
you are missing 'requirements' section, requests lib is not part of stdlib in python

Reply:
Good point. I will change the `requests` library to the url lib provided with ansible. 

I initially wrote it in this way as I looked at another plugins, and they have the similar import behavior. But I agree we should use as few external library as possible.

Reply:
Hey @bcoca, I updated this PR to use the url library shipped with ansible. Could you please review this PR once again? Thanks!